### DISCUSSION TITLE
Size compensation and inward/outward meaning

### DISCUSSION THREAD

**Discussion by jbuggie** [supermerill/SuperSlicer#3008] [Q&A]
I'm confused by the wording here. For XY compensation, inward (-) means to shrink the layer and outward (+) to enlarge. Yet when it comes to vertical hole size compensation, it seems to imply that inward (-) makes the holes larger and outward (+) smaller. The popup helps further said that the number here would be added to the inner XY compensation.  So if I want to make the object overall size +0.1mm larger and the holes +0.25mm larger, what should I put in?

Currently, I set outter (0.1), inner (0.1), and vertical hole size comp (-0.25). So if  vertical hole size comp (-0.25) is added to inner XY compensation (0.1), would the holes end up only 0.15mm larger?

**Comment by Ahmed-mmm** [comment]
if you want to make the object overall size +0.1mm larger and the holes +0.25mm larger.
DO
XY compensation Outer : +0.1
vertical hole shrinking compensation : +0.25
![Screenshot ](https://user-images.githubusercontent.com/106821318/182026551-de33eac2-72f3-4169-94f6-d68a83a05890.jpg)
